### DISCUSSION TITLE
dependencies for contributor template

### DISCUSSION THREAD

**Discussion by benwandrew** [AutoResearch/autora#396] [Core Contributions]
if we're going to rely upon / reference `torch` for seeding tests in our template (https://github.com/AutoResearch/autora-template/blob/main/tests/README.md), shall we just add it to the default dependencies in the `pyproject.toml` for the template as well?

**Comment by musslick** [comment]
No, definitely not. Avoiding the torch dependency was one of the reasons for restructuring. We could comment the relevant part, stating that this is recommended in case someone wanted to use torch...
